src/spells1.cc

```diff
@@ -3324,14 +3258,7 @@ static bool_ project_f(int who, int r, int y, int x, int dam, int typ)
 				}
 
 				/* Forget the door */
-				c_ptr->info &= ~(CAVE_MARK | CAVE_TRDT);
-
-				/* Remove normal traps */
-				c_ptr->t_idx = 0;
-
-				/* Disarm monster traps */
-				if (c_ptr->feat == FEAT_MON_TRAP)
-					c_ptr->special = c_ptr->special2 = 0;
```

Comment:
Can "special" and "special2" be removed? (I suspect not, might be used for other things.)